components/button/button.css

```diff
@@ -0,0 +1,96 @@
+.btn-wrapper {
+  gap: 2rem;
+}
+
+.btn {
+  border: 0;
+  color: white;
+  cursor: pointer;
+  background: white;
```

Comment:
`background` and `color` values should not be the same.

Reply:
Thanks jantu for finding this vulnerability 💯